gnssanalysis/gn_diffaux.py

```diff
@@ -358,12 +358,13 @@ def compare_clk(
             _logging.debug("compare_clk: syncing clk_a_unst with common_svs as not equal")
             clk_a_unst = clk_a_unst[common_svs]
 
-    if len(norm_types) != 0:
-        _logging.info(f":_clk_compare:using {norm_types} clk normalization")
-        if "sv" in norm_types:
-            norm_types[norm_types.index("sv")] = _gn_io.clk.select_norm_svs_per_gnss(
+    norm_types_new = norm_types.copy() # DO NOT overwrite norm_types otherwise it will cause errors when the function is called in a loop
+    if len(norm_types_new) != 0:
+        _logging.info(f":compare_clk: using {norm_types_new} clk normalization")
+        if "sv" in norm_types_new:
```

Comment:
Available options include 'epoch', 'daily', 'sv', any satellite PRN (or a list of PRNs), or any combination of them. 'sv' means it will automatically choose a group of satellites (one satellite per constellation) but users can also name the satellites explicitly. I would say 'sv' is useful depending on what would the users like to do.

Reply:
Thanks for that, that makes a lot more sense